lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,13 +1,13 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
-    <ul id="archives">
-      <% sidebar.archives.each do |month| %>
-        <% counter = sidebar.show_count ? "<em>(#{month[:article_count]})</em>" : "" %>
-        <li>
-          <%= link_to"#{month[:name]} #{counter}".html_safe,  articles_by_month_path( month[:year], "%02i" % (month[:month]+1) ) %>
-        </li>
-      <% end %>
-    </ul>
-  </div>
-<% end %>
+ <h3 class="sidebar-title"><%= sidebar.title %></h3>
```

Comment:
Whoops, that was a fix for Issue 1, I reverted it back to underscores. This commit should only fix Issue 2 and only Issue 2.